schema/table.go

```diff
@@ -250,6 +258,26 @@ func (t *Table) processFields(typ reflect.Type) {
 			t.addUnique(subfield, embfield.prefix, v)
 		}
 	}
+
+	if len(embedded) > 0 && !DisableSortFieldsByStruct {
```

Comment:
@j2gg0s let's remove DisableSortFieldsByStruct and always sort the fields. Are there any undesired consequences?